tests/load/kubernetes-config/locust-master-controller.yml

```diff
@@ -20,12 +20,8 @@ spec:
           env:
             - name: LOCUST_MODE_MASTER
               value: "true"
-            - name: TARGET_HOST
-              value: [TARGET_HOST]
             - name: LOCUST_CSV
               value: [LOCUST_CSV]
-            - name: LOCUST_HOST
-              value: [LOCUST_HOST]
```

Comment:
Do you need to keep `LOCUST_HOST`, maybe by setting it in the `setup_k8s.sh`? I noticed it's in the docker-compose.yml still and normally the env vars should be the same.